lib/screens/widgets/mitgliedStufenPieChart.widget.dart

```diff
@@ -48,8 +48,8 @@ class MitgliedStufenPieChartState extends State<MitgliedStufenPieChart> {
               key, value, Colors.red, 'assets/images/rover.png', index));
           break;
         case "LeiterIn":
-          sectionData.add(createPieElement(
-              key, value, Colors.yellow, 'assets/images/lilie.png', index));
+          sectionData.add(createPieElement(key, value, Colors.yellow,
```

Comment:
Colors.yellow ist doch jetzt die Leiter Farbe aus dem CD der DPSG

Reply:
Oh das habe ich übersehen :+1: . Habe den Fix bisschen ausgeweitet und `Stufe'  und `StufeEnum` in ein enhanced enum verwandelt. Ist jetzt alles cleaner, zentraler und einfach finde ich.